enhancements/observability/multicluster-observability-addon.md

```diff
@@ -0,0 +1,712 @@
+```
+
+In turn the addon will compile a `ManifestWork` for the managed cluster `spoke` as follows and pass it over it's WorkAgentController:
+
+```yaml
+kind: ManifestWork
+metadata:
+  name: addon-multicluster-observability-addon-deploy-0
+  namespace: spoke
+spec:
+  workload:
+    manifests:
+    - apiVersion: project.openshift.io/v1
+      kind: Project
+      metadata:
+        name: openshift-opentelemetry-operator
+    - apiVersion: operators.coreos.com/v1
+      kind: OperatorGroup
+      metadata:
+        name: openshift-opentelemetry-operator-ghl6v
+        namespace: openshift-opentelemetry-operator
+      spec:
+        upgradeStrategy: Default
+    - apiVersion: operators.coreos.com/v1alpha1
+      kind: Subscription
+      metadata:
+        name: opentelemetry-product
+        namespace: openshift-opentelemetry-operator
+      spec:
+        channel: stable
+        installPlanApproval: Automatic
+        name: opentelemetry-product
+        source: redhat-operators
+        sourceNamespace: openshift-marketplace
+        startingCSV: opentelemetry-operator.v0.81.1-5
+    - apiVersion: v1
+      kind: Secret
+      metadata:
+        name: tracing-otlp-auth
+        namespace: spoke
+      stringData:
+        'tls.crt': "TLS client certificate"
+        'tls.key': "TLS key"
+        'ca-bundle.crt': "Certificate Authority certificate"
+    - apiVersion: opentelemetry.io/v1alpha1
+      kind: OpenTelemetryCollector
+      metadata:
+        name: spoke-otelcol
+        namespace: open-cluster-management
+      spec:
+        config: |
+          receivers:
+            jaeger:
+              protocols:
+                grpc:
+            otlp:
+              protocols:
+                grpc:
+                http:
+          processors:
+          exporters:
+            debug:
+            otlp:
+              tls:
+                insecure: true
+          service:
+            pipelines:
+              traces:
+                receivers: [jaeger,otlp]
+                processors: []
+                exporters: [otlp, debug]
+```
+
+
+#### Hypershift [optional]
+
+TBD
+
+### Drawbacks
+
+TBD
+
+## Design Details
+
+### Open Questions [optional]
+
+The [addon-framework](ocm-addon-framework) is still lacking some features that will fully enable the behaviours described in this proposal namelly: 
+- [ ] Support for reconcile resources generated by the addon (Jira issues [LOG-5343](https://issues.redhat.com/browse/LOG-5343) and [TRACING-4148](https://issues.redhat.com/browse/TRACING-4148)). MCOA needs this otherwise it will not react on changes to those resources.
+- [ ] Support for multiple resources of the same kind in the ManagedClusterAddOn resource (Jira issues [LOG-5344](https://issues.redhat.com/browse/LOG-5344)). MCOA needs this otherwise it will not react on changes to those resources.
+
+### Test Plan
+
+TBD
+
+### Graduation Criteria
+
+TBD
+
+#### Dev Preview -> Tech Preview
+
+TBD
+
+#### Tech Preview -> GA
+
+TBD
```

Comment:
It's time to outline the graduation criteria here, short listed as:

1. Dev Preview: CLF/OTelCollector CRs, mTLS auth & static auth, Per-ClusterSet Forwarding.
2. Tech-Preview: Refined CLF/OTelCollector CRs usage, managed auth types, HealthChecking per Signal
3. GA: TBD